eth-balance/src/pb/messari.token.v1.rs

```diff
@@ -0,0 +1,217 @@
+// @generated
+#[derive(Clone, PartialEq, ::prost::Message)]
+pub struct Tokens {
```

Comment:
This file should not be in the PR - Use build.rs to push this to target folder like the other substreams projects do